spotifycli/spotifycli.py

```diff
@@ -92,7 +113,10 @@ def get_arguments():
     return [
         ("--version", "shows version number"),
         ("--status", "shows song name and artist"),
-        ("--statusposition", "shows song name and artist, with current playback position"),
+        (
+            "--statusposition",
+            "shows song name and artist, with current playback position"
+        ),
```

Comment:
Line wrapped for format, happy to revert this if you are OK with longer lines.

Reply:
You can format it in the same way as other options in one line. One line is longer, bo overall format is more consistent.

Reply:
I think you are saying you'd like it kept in one long line rather than broken up. That is fine, but that would require telling `pycodestyle` to allow longer lines.  

For now I will leave this line wrapped so that the format-checker passes.  If you want me to adjust the rules on the format checker so that longer lines are OK, I'll do that.